ShowSQLTables/DataTaker.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Data.SqlClient;
+
+namespace ShowSQLTables
+{
+    class DataTable
+    {
+        private SqlCommand command;
+        private SqlDataReader reader;
+
+        public DataTable(SqlCommand command, SqlDataReader reader)
+        {
+            this.command = command;
+            this.reader = reader;
+        }
+
+        public void DataTaker()
+        {
+            if (reader.HasRows)
+            {
+                for (int i = 0; i < reader.FieldCount; i++)
+                {
+                    Console.Write("{0} \t", reader.GetName(i));
+                }
+                Console.WriteLine();
+
+                while (reader.Read())
+                {
+                    int num = 0;
+                    while (num < reader.FieldCount)
+                    {
+                        object property = reader.GetValue(num);
+
+                        Console.Write("{0} \t", property);
```

Comment:
Dont print results but return them so the code outside can use the results for other sake rather then only printing. Think about reusability of this class